app/models/video.rb

```diff
@@ -4,6 +4,7 @@ class Video < ActiveRecord::Base
 
   attr_accessible :description, :title, :youtube_id, :starred, :duration
   acts_as_taggable
+  acts_as_taggable_on :category
```

Comment:
Simplicity. Categories don't have much value as a model, and outsourcing most of the database stuff is cleaner.